terraform/modules/paas/variables.tf

```diff
@@ -45,8 +52,10 @@ locals {
   app_name_suffix              = var.app_environment != "review" ? var.app_environment : "pr-${var.web_app_host_name}"
   web_app_name                 = "publish-teacher-training-${local.app_name_suffix}"
   publish_app_name             = "publish-${local.app_name_suffix}"
-  find2_app_name               = var.app_environment == "review" ? "find2-${local.app_name_suffix}" : ""
-  cloudapp_names               = var.app_environment == "review" ? [local.web_app_name, local.publish_app_name, local.find2_app_name] : [local.web_app_name, local.publish_app_name]
+#  find2_app_name               = var.app_environment == "review" ? "find2-${local.app_name_suffix}" : ""
+  cloudapp_names               = var.app_environment == "review" ? [local.web_app_name, local.publish_app_name, "find2-${local.app_name_suffix}"] : [local.web_app_name, local.publish_app_name]
+  find_app_name                = "find-${local.app_name_suffix}"
+  find_cloudapp_names          = var.find_route_target != null ? "find-${local.app_name_suffix}" : null
```

Comment:
As discussed, this extra variable is not required

Reply:
removed